models/project_issues.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+// ProjectIssues saves relation from issue to a project
+type ProjectIssues struct {
+	ID        int64 `xorm:"pk autoincr"`
+	IssueID   int64 `xorm:"INDEX"`
+	ProjectID int64 `xorm:"INDEX"`
+
+	// If 0, then it has not been added to a specific board in the project
+	ProjectBoardID int64 `xorm:"INDEX"`
+}
+
+func deleteProjectIssuesByProjectID(e Engine, projectID int64) error {
+	_, err := e.Where("project_id=?", projectID).Delete(&ProjectIssues{})
+	return err
+}
+
+//  ___
+// |_ _|___ ___ _   _  ___
+//  | |/ __/ __| | | |/ _ \
+//  | |\__ \__ \ |_| |  __/
+// |___|___/___/\__,_|\___|
+
+// LoadProject load the project the issue was assigned to
+func (i *Issue) LoadProject() (err error) {
+	return i.loadProject(x)
+}
+
+func (i *Issue) loadProject(e Engine) (err error) {
+	if i.Project == nil {
+		var p Project
+		if _, err = e.Table("project").
+			Join("INNER", "project_issues", "project.id=project_issues.project_id").
+			Get(&p); err != nil {
```

Comment:
I think this will fix the problem

```suggestion
			Join("INNER", "project_issues", "project.id=project_issues.project_id").
			Where("project_issues.issue_id = ?", i.ID).
			Get(&p); err != nil {
```

Reply:
:sweat_smile: O it was my fault - how did I missed that